android/src/main/java/com/imagepicker/VideoMetadata.java

```diff
@@ -0,0 +1,28 @@
+package com.imagepicker;
+
+import static java.lang.Integer.parseInt;
+
+import android.content.Context;
+import android.media.MediaMetadataRetriever;
+import android.net.Uri;
+
+public class VideoMetadata {
+  int duration;
+  int bitrate;
+
+  public VideoMetadata(Uri uri, Context context) {
+    MediaMetadataRetriever m = new MediaMetadataRetriever();
+    m.setDataSource(context, uri);
+    this.duration = Math.round(Float.parseFloat(m.extractMetadata(MediaMetadataRetriever.METADATA_KEY_DURATION))) / 1000;
+    this.bitrate = parseInt(m.extractMetadata(MediaMetadataRetriever.METADATA_KEY_BITRATE));
+    m.release();
```

Comment:
I'd suggest breaking this down a bit to help with readability:

```suggestion
    String duration = metadataRetriever.extractMetadata(MediaMetadataRetriever.METADATA_KEY_DURATION);
    String bitrate = metadataRetriever.extractMetadata(MediaMetadataRetriever.METADATA_KEY_BITRATE);
    // Extract anymore metadata here...

    this.duration = Math.round(Float.parseFloat(duration)) / 1000;
    this.bitrate = parseInt(bitrate);
    metadataRetriever.release();
```

Reply:
thanks. agree! committed your changes 😎